files/sensu_cleanup_aws_clients.rb

```diff
@@ -0,0 +1,221 @@
+#!/usr/bin/env ruby
```

Comment:
Although this is an open source project, you won't be able to use the system ruby (probably)

Our puppet ruby has the aws sdk. Alternatively you could install the aws-sdk using sensu's omnibus ruby (following the pattern of the other handlers)